roundcubemail.spec

```diff
@@ -1,6 +1,6 @@
-%define rcm_version 1.4.13
+%define rcm_version 1.4.15
 Name:           roundcubemail
-Version: 1.4.13
+Version: 1.4.15
```

Comment:
Please revert the change to this line, we are going to bump the release after the merge

Reply:
yep